scripts/codegen.js

```diff
@@ -0,0 +1,8 @@
+const dotenv = require("dotenv");
+
+// Support self-signed certificate of vc-storefront
+process.env.NODE_TLS_REJECT_UNAUTHORIZED="0"
+
```

Comment:
rename all codegen to GQL_schema_generator or related

Reply:
Renamed